src/picongpu/include/simulation_defines/param/speciesAttributes.param

```diff
@@ -45,131 +47,170 @@
-
-/** masking a particle for radiation
- *
- * The mask is used by the user defined filter
- * `RadiationParticleFilter` in `radiation.param` to (de)select
- * particles for the radiation calculation.
- */
-value_identifier(bool, radiationMask, false);
-
-/** number of electrons bound to the atom / ion
- *
- * value type is float_X to avoid casts during the runtime
- * - float_X instead of integer types are reasonable because effective charge
- *   numbers are possible
- * - required for ion species if ionization is enabled
- *
- * \todo connect default to proton number
- */
-value_identifier(float_X,boundElectrons,float_X(0.0));
-
-/** atomic superconfiguration
- *
- * atomic configuration of an ion for collisional-radiative modeling
- *
- * \see flylite.param
- */
-value_identifier(
-    flylite::Superconfig,
-    superconfig,
-    flylite::Superconfig::create( 0. )
-);
-
-/** Total cell index of a particle.
- *
- *  The total cell index is a
- *  N-dimensional DataSpace given by a GPU's
- *    globalDomain.offset + localDomain.offset
- *  added to the N-dimensional cell index the particle belongs to on that GPU.
- */
-value_identifier(DataSpace<simDim>, totalCellIdx, DataSpace<simDim>());
-
-/*! alias for particle shape @see species.param */
-alias(shape);
-
-/*! alias for particle pusher @see species.param */
-alias(particlePusher);
-
-/*! alias for particle ionizers @see ionizer.param */
-alias(ionizers);
-
-/*! alias for ionization energy container @see ionizationEnergies.param */
-alias(ionizationEnergies);
-
-/*! alias for synchrotronPhotons @see speciesDefinition.param */
-alias(synchrotronPhotons)
-
-/*! alias for ion species used for bremsstrahlung */
-alias(bremsstrahlungIons);
-
-/*! alias for photon species used for bremsstrahlung */
-alias(bremsstrahlungPhotons);
-
-/*! alias for particle to field interpolation @see species.param */
-alias(interpolation);
-
-/*! alias for particle current solver @see species.param */
-alias(current);
-
-/*! alias for particle flag: atomic numbers @see ionizer.param
- * - only reasonable for atoms / ions / nuclei
- */
-alias(atomicNumbers);
-
-/*! alias for particle flag: effective nuclear charge @see ionizer.param
- * - only reasonable for atoms / ions / nuclei
- */
-alias(effectiveNuclearCharge);
-
-/*! alias for particle population kinetics model, e.g. FLYlite
- * @see flylite.param
- */
-alias(populationKinetics);
-
-/*! alias for particle mass ratio
- *
- * mass ratio between base particle @see `speciesConstants.param` SI::BASE_MASS_SI
- * and a user defined species
- *
- * default value: 1.0 if unset
- */
-alias(massRatio);
-
-/*! alias for particle charge ratio
- *
- * charge ratio between base particle @see `speciesConstants.param` SI::BASE_CHARGE_SI
- * and a user defined species
- *
- * default value: 1.0 if unset
- */
-alias(chargeRatio);
-
-/*! alias for particle density ratio
- *
- * density ratio between default density @see `density.param` SI::BASE_DENSITY_SI
```

Comment:
- wrong usage of `@see`
- wrong formatting of file name
- missing verbatim formatting of var
- note: doxygen linkes known names/files/classes automatically, without `@see`